### PR TITLE
[WIP] templates restructure

### LINKED ISSUE #470: Standardize values.yaml
### Is your feature request related to a problem? Please describe.

- When we deploy to kubernetes we want to modify some of the configurations like filters. inputs and outputs but still use the fluentbit installation enabled by the Kubernetes: true setting.
- values.yaml need some linting
- Some standardizing like changing enable to enabled should be conscidered to follow comunity-practices
- default variables should be enabled in template and uncommented when possible
  - default disabled components should have all settings uncommented

### Describe the solution you'd like

- Yaml-linted according to https://yamllint.readthedocs.io/en/stable/ 
- Enable is changed to Enabled
- All inputs, filters and outputs can be enabled/disabled

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by havardelnan**
### What this PR does / why we need it:

The PR makes the template more uniform and compliant to community practices

### Which issue(s) this PR fixes:

Fixes #470

### Does this PR introduced a user-facing change?

```release-note
- Kubernetes root key is removed in a backward compatible manner. Use .fluentbit.enabled instead.


action required
nodeExporterMetrics
  - Move config thats not implemented to  "additionalConfig" set enabled to true
    before:
      nodeExporterMetrics: 
        tag: node_metrics //Implemented in the chart
        someunimplementedsetting: false
        anotherunimplementedsetting: test
    after:
      nodeExporterMetrics: 
        enabled: true
        tag: node_metrics
        additionalConfig:
          someunimplementedsetting: false
          anotherunimplementedsetting: test
Opentelemetry and Opensearch needs to be changed acordingly:
    before:
      Open**** : 
        somsetting: false
       anothersetting: test
    after:
      Open*** : 
        enabled: true
        additionalConfig:
          somesetting: false
          anothersetting: test
```

### Additional documentation, usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by havardelnan** [comment]
I realize this PR is somewhat opinionated, all feedback is appreciated

**Comment by benjaminhuo** [comment]
> I realize this PR is somewhat opinionated, all feedback are appreciated

I like it :)

File: charts/fluent-operator/templates/fluentbit-clusterinput-systemd.yaml [hunk 1]
```diff
@@ -8,11 +9,13 @@ metadata:
...
-    systemdFilter:
-      - _SYSTEMD_UNIT={{ .Values.containerRuntime }}.service
-      - _SYSTEMD_UNIT=kubelet.service
-{{- end }}
\ No newline at end of file
+    tag: {{ .Values.fluentbit.input.systemd.tag | default "service.*" }}
+    path: {{ .Values.fluentbit.input.systemd.path | default "/var/log/journal" }}
+    db: {{ .Values.fluentbit.input.systemd.db | default "/fluent-bit/tail/systemd.db" }}
+    dbSync: {{ .Values.fluentbit.input.systemd.dbSync | default "Normal" }}
+    systemdFilter: {{ .Values.fluentbit.input.systemd.systemdFilter | default ["_SYSTEMD_UNIT={{ .Values.containerRuntime }}.service", "_SYSTEMD_UNIT=kubelet.service]" }}
```
Comment (wenchajun) [inline charts/fluent-operator/templates/fluentbit-clusterinput-systemd.yaml:16]: This seems to have been executed with an error.

Comment (havardelnan) [reply charts/fluent-operator/templates/fluentbit-clusterinput-systemd.yaml]: Changed this and moved defaults back to values-file

**Comment by benjaminhuo** [comment]
Close it for now because of inactive

**Pull request closed without merging**